service/Cargo.toml

```diff
@@ -12,6 +12,7 @@ doctest = false
 anyhow = "1.0.57"
 async-trait = "0.1.56"
 axum = { version = "0.5.7", optional = true }
+bincode = "1.2.1"
```

Comment:
This crate should be made optional and only be included with the `persist` feature